search/elastic.py

```diff
@@ -58,6 +58,13 @@ def translate_facet(result):
     if "facets" in es_response:
         response["facets"] = {facet: translate_facet(es_response["facets"][facet]) for facet in es_response["facets"]}
 
+    if "aggregations" in es_response and "total_records" in es_response["aggregations"]:
+        # Total number without Filters
+        response["doc_count"] = es_response["aggregations"]["total_records"]["doc_count"]
+        if "filtered_id" in es_response["aggregations"]["total_records"]:
+            # For some low level Roles: counting for specified course_keys / program_uuids
+            response["doc_count"] = es_response["aggregations"]["total_records"]["filtered_id"]["doc_count"]
```

Comment:
- We take `aggregations. total_records. filtered_id. doc_count` as `"doc_count"` if this field exists.